packages/sequelize-model/src/index.ts

```diff
@@ -1,14 +1,13 @@
 import sqOriginal from './sequelizeInit';
```

Comment:
you just call it sq and re export it

Reply:
I can but I prefer to keep the same semantic for all the imports / exports.